tests/cucumber/steps/steps.js

```diff
@@ -459,18 +459,13 @@ module.exports = function getSteps(options) {
     async function (amt, note) {
       [this.pk] = this.accounts;
       const result = await this.v2Client.getTransactionParams().do();
-      this.lastRound = result.lastRound;
-      this.txn = {
+      this.txn = algosdk.makePaymentTxnWithSuggestedParamsFromObject({
```

Comment:
Uses the canonical factory method to build up the `Transaction` object instead of relying on duck typing to infer the object type.